ngraph_bridge/ngraph_builder.cc

```diff
@@ -91,15 +92,6 @@ static void SaveNgOp(Builder::OpMap& ng_op_map, const std::string& op_name,
   ng_op_map[op_name].push_back(output_node);
 }
 
-template <class TOpType, class... TArg>
-std::shared_ptr<TOpType> ConstructNgNode(const std::string& op_name,
```

Comment:
this standalone function is now moved inside `Builder` class as a static function